Create Releasable Package Drop is Failing for release-4.0

https://github.com/microsoft/TypeScript/actions?query=workflow%3A%22Create+Releasable+Package+Drop%22

```
npm ERR! code ENOSELF
npm ERR! Refusing to install package with name "typescript" under a package
npm ERR! also called "typescript". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR! 
```

[First failure starts here, though it's not clear why](https://github.com/microsoft/TypeScript/actions/runs/226162596).

--- a/.github/workflows/new-release-branch.yaml
+++ b/.github/workflows/new-release-branch.yaml
@@ -23,7 +23,7 @@ jobs:
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/typescript.d.ts
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/tsserverlibrary.d.ts
         sed -i -e 's/const version = `${versionMajorMinor}.0-.*`/const version = `${versionMajorMinor}.0-${{ github.event.client_payload.core_tag || 'dev' }}`/g' src/compiler/corePublic.ts
-        npm install
+        npm ci
         gulp LKG
         npm test
         git diff
@@ -32,7 +32,7 @@ jobs:
         git add tests/baselines/reference/api/typescript.d.ts
         git add tests/baselines/reference/api/tsserverlibrary.d.ts
         git add ./lib
-        git config user.email "ts_bot@rcavanaugh.com"
+        git config user.email "typescriptbot@microsoft.com"
         git config user.name "TypeScript Bot"
         git commit -m 'Bump version to ${{ github.event.client_payload.package_version }} and LKG'
         git push --set-upstream origin ${{ github.event.client_payload.branch_name }}

--- a/.github/workflows/release-branch-artifact.yaml
+++ b/.github/workflows/release-branch-artifact.yaml
@@ -21,8 +21,7 @@ jobs:
         npm uninstall tslint --no-save
     - name: npm install and test
       run: |
-        npm install
-        npm update
+        npm ci
         npm test
       env:
         CI: true
@@ -41,4 +40,3 @@ jobs:
       with:
         name: tgz
         path: typescript.tgz
-    

--- a/.github/workflows/set-version.yaml
+++ b/.github/workflows/set-version.yaml
@@ -29,7 +29,7 @@ jobs:
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/typescript.d.ts
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/tsserverlibrary.d.ts
         sed -i -e 's/const version = .*;/const version = "${{ github.event.client_payload.package_version }}" as string;/g' src/compiler/corePublic.ts
-        npm install
+        npm ci
         gulp LKG
         npm test
         git diff
@@ -38,7 +38,7 @@ jobs:
         git add tests/baselines/reference/api/typescript.d.ts
         git add tests/baselines/reference/api/tsserverlibrary.d.ts
         git add ./lib
-        git config user.email "ts_bot@rcavanaugh.com"
+        git config user.email "typescriptbot@microsoft.com"
         git config user.name "TypeScript Bot"
         git commit -m 'Bump version to ${{ github.event.client_payload.package_version }} and LKG'
         git push

--- a/.github/workflows/sync-branch.yaml
+++ b/.github/workflows/sync-branch.yaml
@@ -20,10 +20,10 @@ jobs:
     # required client_payload members:
     # branch_name - the target branch
     - run: |
-        git config user.email "ts_bot@rcavanaugh.com"
+        git config user.email "typescriptbot@microsoft.com"
         git config user.name "TypeScript Bot"
         git fetch origin master
         git merge master --no-ff
-        npm install
+        npm ci
         npm test
         git push

--- a/.github/workflows/update-lkg.yml
+++ b/.github/workflows/update-lkg.yml
@@ -17,9 +17,9 @@ jobs:
 
     - name: Configure Git and Update LKG
       run: |
-        git config user.email "ts_bot@rcavanaugh.com"
+        git config user.email "typescriptbot@microsoft.com"
         git config user.name "TypeScript Bot"
-        npm install
+        npm ci
         gulp LKG
         npm test
         git diff

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,6 +2,6 @@
 FROM node:current
 COPY . /typescript
 WORKDIR /typescript
-RUN npm install
+RUN npm ci
 RUN npm i -g gulp-cli
 RUN gulp configure-insiders && gulp LKG && gulp clean && npm pack .

--- a/README.md
+++ b/README.md
@@ -67,7 +67,7 @@ Install [Gulp](https://gulpjs.com/) tools and dev dependencies:
 
 ```bash
 npm install -g gulp
-npm install
+npm ci
 ```
 
 Use one of the following to build and test:

